crates/kumod/src/http_server/inject_v1.rs

```diff
@@ -1232,4 +1232,86 @@ Ok(
 "#
         );
     }
+
+    #[tokio::test]
```

Comment:
FWIW, these newly added tests in this file don't (shouldn't, AFAICT) be affected by any compression settings, as compression is a transport thing and these tests are directly creating the structs that the requests are parsed into.

I don't think we need these, as I don't think they are exercising anything different from the other tests?

Reply:
You are spot on. I've removed those. I think the integration tests are sufficient.